### PR TITLE
Update Japanese locale

### PR DESCRIPTION
**Pull request by hiro0218**
Hi

> Rectangle v0.48
> Added the ability to move the cursor along with the window when moving across displays, along with a toggle in the preferences window.

translated it into Japanese.

### CODE REVIEW

**Comment by rxhanson** [comment]
Thanks!

If you don't mind, would you like to translate the following lines as well? That would complete the missing translations for this round.

> /* Class = "NSTextFieldCell"; title = "First Three Fourths"; ObjectID = "T9Z-QF-gwc"; */
> "T9Z-QF-gwc.title" = "";
> 
> /* Class = "NSTextFieldCell"; title = "Last Three Fourths"; ObjectID = "nwX-h6-fwm"; */
> "nwX-h6-fwm.title" = "";
> 
> /* Class = "NSButtonCell"; title = "Import"; ObjectID = "RgZ-Jw-XQZ"; */
> "RgZ-Jw-XQZ.title" = "";
> 
> /* Class = "NSButtonCell"; title = "Export"; ObjectID = "IO3-Hi-7gC"; */
> "IO3-Hi-7gC.title" = "";

If not, that's ok and I will go ahead and merge what's in this PR.

**Comment by hiro0218** [comment]
@rxhanson 
I've added translation support. Please check it out. 😉

File: Rectangle/ja.lproj/Main.strings [hunk 1]
```diff
@@ -585,10 +585,10 @@
 "Fko-xs-gN5.title" = "左から2番目";
 
 /* Class = "NSTextFieldCell"; title = "First Three Fourths"; ObjectID = "T9Z-QF-gwc"; */
-"T9Z-QF-gwc.title" = "";
+"T9Z-QF-gwc.title" = "4/3にして左寄せ";
```
Comment (ryonakano) [inline Rectangle/ja.lproj/Main.strings:588]:
```suggestion
"T9Z-QF-gwc.title" = "3/4にして左寄せ";
```

画面の4分の3に分割するということなので、"3/4"だと思います。

File: Rectangle/ja.lproj/Main.strings [hunk 2]
```diff
@@ -585,10 +585,10 @@
...
 
 /* Class = "NSTextFieldCell"; title = "Last Three Fourths"; ObjectID = "nwX-h6-fwm"; */
-"nwX-h6-fwm.title" = "";
+"nwX-h6-fwm.title" = "4/3にして右寄せ";
```
Comment (ryonakano) [inline Rectangle/ja.lproj/Main.strings:591]:
```suggestion
"nwX-h6-fwm.title" = "3/4にして右寄せ";
```

File: Rectangle/ja.lproj/Main.strings [hunk 3]
```diff
@@ -641,10 +641,10 @@
...
-"RgZ-Jw-XQZ.title" = "";
+"RgZ-Jw-XQZ.title" = "インポート";
 
 /* Class = "NSButtonCell"; title = "Export"; ObjectID = "IO3-Hi-7gC"; */
-"IO3-Hi-7gC.title" = "";
+"IO3-Hi-7gC.title" = "エクスポート";
 
 /* Class = "NSButtonCell"; title = "Move cursor along with window across displays"; ObjectID = "Pbz-DF-hgG"; */
-"Pbz-DF-hgG.title" = "Move cursor along with window across displays";
+"Pbz-DF-hgG.title" = "ディスプレイ上のウィンドウに沿ってカーソルを移動する";
```
Comment (ryonakano) [inline Rectangle/ja.lproj/Main.strings:650]:
```suggestion
"Pbz-DF-hgG.title" = "別ディスプレイへのウィンドウ移動時にカーソルも一緒に移動させる";
```

「ウィンドウに沿って」だと、意味が伝わりにくいと思います。[こちらの記事](https://applech2.com/archives/20210704-rectangle-for-mac-move-cursor-across-dispalys.html)もご覧ください。

**Review by ryonakano — CHANGES_REQUESTED** [review]
@hiro0218 Thank you for completing Japanese translation! I reviewed your translation suggestion.

File: Rectangle/ja.lproj/Main.strings (hunk 1 above)
Comment (hiro0218) [reply Rectangle/ja.lproj/Main.strings:588]: 「4分の3」を思い浮かべて「4/3」としてしまいました。訂正ありがとうございます。

File: Rectangle/ja.lproj/Main.strings (hunk 3 above)
Comment (hiro0218) [reply Rectangle/ja.lproj/Main.strings:650]: 採用させて頂きます🙏

**Comment by hiro0218** [comment]
@ryonakano Thank you for your review!

**Review by ryonakano — APPROVED** [review]
LGTM!

**Comment by rxhanson** [comment]
Great, thanks!

**Pull request merged**